org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/InitializeLaunchConfigurations.java

```diff
@@ -58,6 +58,13 @@ public static String getVSCodeLocation(String appendPathSuffix) {
 	}
 
 	public static String getNodeJsLocation() {
+		{
```

Comment:
I personally don't mind extra curly braces; it's a good way to make sure some local variable don't leak to undesired parts of the code.

Reply:
If you think it's better then we can keep extra curly braces.

Reply:
I don't care enough, I leave you the final word and the final steps of the review if it's ok with you ;)

Reply:
Hi,

I use those extra braces a lot in my own codes. Is there a coding style document I can follow? Maybe something that is general to all Eclipse contribution? Also, I am worried about the formatting styles (I use Eclipse default). In bigger changes, it could be a problem.

Reply:
AFAIK, there is no coding style document, and I'm personally against enforcing coding styles in general, even in the same project. They're more frequently a PITA than an assistance.